frontend/src/components/daemonset/Details.tsx

```diff
@@ -29,6 +29,10 @@ export default function DaemonSetDetails() {
             name: t('Selector'),
             value: <MetadataDictGrid dict={item.spec.selector.matchLabels || {}} />,
           },
+          {
+            name: t('Node Selector'),
```

Comment:
Thanks, While testing this I found a issue with the DaemonSet details view looks like the back button is broken there it should take us to the list view but it right now takes us to the workload overview view

Reply:
can we add a fix for that as well in this PR itself??

Reply:
On further testing looks like the back button is broken for 
```DaemonSets, StatefulSets, Jobs, CronJobs```